koku/api/currency/models.py

```diff
@@ -0,0 +1,45 @@
+#
+# Copyright 2021 Red Hat Inc.
+# SPDX-License-Identifier: Apache-2.0
+#
+"""Models for AWS cost entry tables."""
+# from uuid import uuid4
+from django.db import models
+from django.db.models import JSONField
+
+
+class ExchangeRates(models.Model):
+    AUD = "aud"
+    CAD = "cad"
+    CHF = "chf"
+    CNY = "cny"
+    DKK = "dkk"
+    EUR = "eur"
+    GBP = "gbp"
+    HKD = "hkd"
+    JPY = "jpy"
+    NOK = "nok"
+    NZD = "nzd"
+    SEK = "sek"
+    SGD = "sgd"
+    USD = "usd"
+    ZAR = "zar"
+    SUPPORTED_CURRENCIES = (
+        (AUD, "AUD"),
+        (CAD, "CAD"),
+        (CHF, "CHF"),
+        (CNY, "CNY"),
+        (DKK, "DKK"),
+        (EUR, "EUR"),
+        (GBP, "GBP"),
+        (HKD, "HKD"),
+        (JPY, "JPY"),
+        (NOK, "NOK"),
+        (NZD, "NZD"),
+        (SEK, "SEK"),
+        (SGD, "SGD"),
+        (USD, "USD"),
+        (ZAR, "ZAR"),
+    )
+    baseCurrency = models.CharField(max_length=100, choices=SUPPORTED_CURRENCIES, unique=True)
```

Comment:
Real quick, just for Python-y consistency, you can name these like `base_currency`. 

This table probably needs three columns. The base currency, the currency you are converting to, and the exchange rate.

Reply:
Yep! These have been changed to the naming convention. There is now a third column for target_currency.